### ISSUE TITLE
Test: TableComponent component

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#9191]
## Description
The TableComponent component (`src/components/paragraph/table/index.tsx`) should have tests, per #8970.

## Acceptance Criteria
- [ ] The component has automated tests written per [guidelines](https://github.com/department-of-veterans-affairs/next-build/blob/main/READMEs/testing.md).

## Implementation

- Tests already exist on the design side, testing within this ticket should not be extensive

**Comment by ndouglas** [comment]
@amponce Is it normal/semi-expected for http://localhost:3000/demo/table to be blank right now?

I see this:
```html


  | <!DOCTYPE html><html lang="en" dir="ltr"><head><style data-next-hide-fouc="true">body{display:none}</style><noscript data-next-hide-fouc="true"><style>body{display:block}</style></noscript><meta charSet="utf-8"/><meta name="viewport" content="width=device-width"/><meta name="next-head-count" content="2"/><meta charSet="utf-8"/><link rel="stylesheet" data-entry-name="style.css" href="https://unpkg.com/@department-of-veterans-affairs/formation/dist/formation.min.css"/><link rel="stylesheet" data-entry-name="style.css" href="https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/generated/style.css"/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/_next/static/chunks/polyfills.js?ts=1655907722867"></script><script src="/_next/static/chunks/webpack.js?ts=1655907722867" defer=""></script><script src="/_next/static/chunks/main.js?ts=1655907722867" defer=""></script><script src="/_next/static/chunks/pages/_app.js?ts=1655907722867" defer=""></script><script src="/_next/static/chunks/pages/demo/table.js?ts=1655907722867" defer=""></script><script src="/_next/static/development/_buildManifest.js?ts=1655907722867" defer=""></script><script src="/_next/static/development/_ssgManifest.js?ts=1655907722867" defer=""></script><script src="/_next/static/development/_middlewareManifest.js?ts=1655907722867" defer=""></script><noscript id="__next_css__DO_NOT_USE__"></noscript></head><body><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=undefined" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript><div id="__next"></div><script src="/_next/static/chunks/react-refresh.js?ts=1655907722867"></script><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"data":{"type":"paragraph--table","id":"fe33b703-5ff0-4681-93fc-bc46d44ffbba","drupal_internal__id":4604,"drupal_internal__revision_id":22942,"langcode":"en","status":true,"created":"2019-07-12T16:47:43+00:00","parent_id":"4603","parent_type":"paragraph","parent_field_name":"field_va_paragraphs","behavior_settings":[],"default_langcode":true,"revision_translation_affected":null,"field_table":{"value":[["\u003cstrong\u003eSchool name\u003c/strong\u003e","\u003cstrong\u003eVSOC counselor email address\u003c/strong\u003e","\u003cstrong\u003eRegional office\u003c/strong\u003e"],["George Mason University","Valerie Smith \u003ca href=\"mailto:Valerie.Smith11@va.gov\"\u003eValerie.Smith11@va.gov\u003c/a\u003e","National Capital Region Benefits Office (372)"],["Northern Virginia Community College - Alexandria","Michelle Pohmer \u003ca href=\"mailto:Michelle.Pohmer@va.gov\"\u003eMichelle.Pohmer@va.gov\u003c/a\u003e","National Capital Region Benefits Office (372)"],["Northern Virginia Community College - Annandale","Michelle Pohmer \u003ca href=\"mailto:Michelle.Pohmer@va.gov\"\u003eMichelle.Pohmer@va.gov\u003c/a\u003e","National Capital Region Benefits Office (372)"],["Norfolk State University","Donecia Lawson \u003ca href=\"mailto:Donecia.Lawson@va.gov\"\u003eDonecia.Lawson@va.gov\u003c/a\u003e","Roanoke (314)"],["Tidewater Community College - Norfolk","Donecia Lawson \u003ca href=\"mailto:Donecia.Lawson@va.gov\"\u003eDonecia.Lawson@va.gov\u003c/a\u003e","Roanoke (314)"],["Tidewater Community College - Chesapeake","Teri Nguyen \u003ca href=\"mailto:Teri.Nguyen1@va.gov\"\u003eTeri.Nguyen1@va.gov\u003c/a\u003e","Roanoke (314)"],["Tidewater Community College - Portsmouth","Teri Nguyen \u003ca href=\"mailto:Teri.Nguyen1@va.gov\"\u003eTeri.Nguyen1@va.gov\u003c/a\u003e","Roanoke (314)"],["Tidewater Community College - Virginia Beach","Carla Riggins \u003ca href=\"mailto:Carla.Riggins@va.gov\"\u003eCarla.Riggins@va.gov\u003c/a\u003e","Roanoke (314)"],["Old Dominion University","Vacant","Roanoke (314)"],["ECPI University","Maureen Gibson \u003ca href=\"mailto:Maureen.Gibson@va.gov\"\u003eMaureen.Gibson@va.gov\u003c/a\u003e","Roanoke (314)"],["Liberty University","Vacant","Roanoke (314)"],["St. Leo University - South Hampton","Saundra Oliver \u003ca href=\"mailto:Saundra.Oliver@va.gov\"\u003eSaundra.Oliver@va.gov\u003c/a\u003e","Roanoke (314)"]],"format":"rich_text","caption":""},"links":{"self":{"href":"https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/paragraph/table/fe33b703-5ff0-4681-93fc-bc46d44ffbba"}},"paragraph_type":{"type":"paragraphs_type--paragraphs_type","id":"80b23186-37b2-4c2b-8223-24e952485df8","resourceIdObjMeta":{"drupal_internal__target_id":"table"}},"relationshipNames":["paragraph_type"]}},"__N_SSG":true},"page":"/demo/table","query":{},"buildId":"development","isFallback":false,"gsp":true,"scriptLoader":[]}</script></body></html>
-- | --


<!DOCTYPE html><html lang="en" dir="ltr"><head><style data-next-hide-fouc="true">body{display:none}</style><noscript data-next-hide-fouc="true"><style>body{display:block}</style></noscript><meta charSet="utf-8"/><meta name="viewport" content="width=device-width"/><meta name="next-head-count" content="2"/><meta charSet="utf-8"/><link rel="stylesheet" data-entry-name="style.css" href="https://unpkg.com/@department-of-veterans-affairs/formation/dist/formation.min.css"/><link rel="stylesheet" data-entry-name="style.css" href="https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/generated/style.css"/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="[/_next/static/chunks/polyfills.js?ts=1655907722867](http://localhost:3000/_next/static/chunks/polyfills.js?ts=1655907722867)"></script><script src="[/_next/static/chunks/webpack.js?ts=1655907722867](http://localhost:3000/_next/static/chunks/webpack.js?ts=1655907722867)" defer=""></script><script src="[/_next/static/chunks/main.js?ts=1655907722867](http://localhost:3000/_next/static/chunks/main.js?ts=1655907722867)" defer=""></script><script src="[/_next/static/chunks/pages/_app.js?ts=1655907722867](http://localhost:3000/_next/static/chunks/pages/_app.js?ts=1655907722867)" defer=""></script><script src="[/_next/static/chunks/pages/demo/table.js?ts=1655907722867](http://localhost:3000/_next/static/chunks/pages/demo/table.js?ts=1655907722867)" defer=""></script><script src="[/_next/static/development/_buildManifest.js?ts=1655907722867](http://localhost:3000/_next/static/development/_buildManifest.js?ts=1655907722867)" defer=""></script><script src="[/_next/static/development/_ssgManifest.js?ts=1655907722867](http://localhost:3000/_next/static/development/_ssgManifest.js?ts=1655907722867)" defer=""></script><script src="[/_next/static/development/_middlewareManifest.js?ts=1655907722867](http://localhost:3000/_next/static/development/_middlewareManifest.js?ts=1655907722867)" defer=""></script><noscript id="__next_css__DO_NOT_USE__"></noscript></head><body><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=undefined" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript><div id="__next"></div><script src="[/_next/static/chunks/react-refresh.js?ts=1655907722867](http://localhost:3000/_next/static/chunks/react-refresh.js?ts=1655907722867)"></script><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"data":{"type":"paragraph--table","id":"fe33b703-5ff0-4681-93fc-bc46d44ffbba","drupal_internal__id":4604,"drupal_internal__revision_id":22942,"langcode":"en","status":true,"created":"2019-07-12T16:47:43+00:00","parent_id":"4603","parent_type":"paragraph","parent_field_name":"field_va_paragraphs","behavior_settings":[],"default_langcode":true,"revision_translation_affected":null,"field_table":{"value":[["\u003cstrong\u003eSchool name\u003c/strong\u003e","\u003cstrong\u003eVSOC counselor email address\u003c/strong\u003e","\u003cstrong\u003eRegional office\u003c/strong\u003e"],["George Mason University","Valerie Smith \u003ca href=\"mailto:Valerie.Smith11@va.gov\"\u003eValerie.Smith11@va.gov\u003c/a\u003e","National Capital Region Benefits Office (372)"],["Northern Virginia Community College - Alexandria","Michelle Pohmer \u003ca href=\"mailto:Michelle.Pohmer@va.gov\"\u003eMichelle.Pohmer@va.gov\u003c/a\u003e","National Capital Region Benefits Office (372)"],["Northern Virginia Community College - Annandale","Michelle Pohmer \u003ca href=\"mailto:Michelle.Pohmer@va.gov\"\u003eMichelle.Pohmer@va.gov\u003c/a\u003e","National Capital Region Benefits Office (372)"],["Norfolk State University","Donecia Lawson \u003ca href=\"mailto:Donecia.Lawson@va.gov\"\u003eDonecia.Lawson@va.gov\u003c/a\u003e","Roanoke (314)"],["Tidewater Community College - Norfolk","Donecia Lawson \u003ca href=\"mailto:Donecia.Lawson@va.gov\"\u003eDonecia.Lawson@va.gov\u003c/a\u003e","Roanoke (314)"],["Tidewater Community College - Chesapeake","Teri Nguyen \u003ca href=\"mailto:Teri.Nguyen1@va.gov\"\u003eTeri.Nguyen1@va.gov\u003c/a\u003e","Roanoke (314)"],["Tidewater Community College - Portsmouth","Teri Nguyen \u003ca href=\"mailto:Teri.Nguyen1@va.gov\"\u003eTeri.Nguyen1@va.gov\u003c/a\u003e","Roanoke (314)"],["Tidewater Community College - Virginia Beach","Carla Riggins \u003ca href=\"mailto:Carla.Riggins@va.gov\"\u003eCarla.Riggins@va.gov\u003c/a\u003e","Roanoke (314)"],["Old Dominion University","Vacant","Roanoke (314)"],["ECPI University","Maureen Gibson \u003ca href=\"mailto:Maureen.Gibson@va.gov\"\u003eMaureen.Gibson@va.gov\u003c/a\u003e","Roanoke (314)"],["Liberty University","Vacant","Roanoke (314)"],["St. Leo University - South Hampton","Saundra Oliver \u003ca href=\"mailto:Saundra.Oliver@va.gov\"\u003eSaundra.Oliver@va.gov\u003c/a\u003e","Roanoke (314)"]],"format":"rich_text","caption":""},"links":{"self":{"href":"https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/paragraph/table/fe33b703-5ff0-4681-93fc-bc46d44ffbba"}},"paragraph_type":{"type":"paragraphs_type--paragraphs_type","id":"80b23186-37b2-4c2b-8223-24e952485df8","resourceIdObjMeta":{"drupal_internal__target_id":"table"}},"relationshipNames":["paragraph_type"]}},"__N_SSG":true},"page":"/demo/table","query":{},"buildId":"development","isFallback":false,"gsp":true,"scriptLoader":[]}</script></body></html>
```

in the source, so I think the data is being retrieved correctly, but I'm not getting any displayed output.  I know this was done some time back and may not be relevant anymore -- I'm just trying to figure out if my local dev is working correctly 🙂

**Comment by ndouglas** [comment]
D'oh, thought I wrote this comment earlier...

cc @timcosgrove 

@amponce and I discussed this during office hours and looked at the [upstream tests](https://github.com/department-of-veterans-affairs/component-library/blob/master/packages/react-components/src/components/Table/Table.unit.spec.jsx) and figure they're sufficient.  

I don't remember for sure, but I don't think you mentioned a need for any particular documentation or anything in this case.  If you'd like me to do anything else to document that this is being adequately tested upstream, I'm happy to do it 🙂 

Closing unless you disagree/wanna discuss in greater depth.  I'll update the title and issue body to reflect.
